radicle-cli/examples/rad-remote.md

```diff
@@ -0,0 +1,12 @@
+`rad remote` can help with managing git's radicle remotes.
+
+Let's take a look at the radicle remotes we have from cloning a project.
+
+```
+$ rad remote -v
+rad rad://z42hL2jL4XNk6K8oHQaSWfMgCL7ji/z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk (fetch)
+z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi    rad://z42hL2jL4XNk6K8oHQaSWfMgCL7ji/z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi (fetch)
+```
+
+The remote `rad` is our fork of the project, and
+'z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi' is the origin's.
```

Comment:
We should add some more of the commands introduced to the command's example

Reply:
yep! lets converge before and have the commands full implemented before